mediawiki/content.md

```diff
@@ -0,0 +1,62 @@
+# What is MediaWiki?
+
+MediaWiki is free and open-source wiki software. Originally developed by Magnus Manske and improved by Lee Daniel Crocker, it runs on many websites, including Wikipedia, Wiktionary and Wikimedia Commons. It is written in the PHP programming language and stores the contents into a database. Like WordPress, which is based on a similar licensing and architecture, it has become the dominant software in its category.
+
+> [wikipedia.org/wiki/MediaWiki](https://en.wikipedia.org/wiki/MediaWiki)
+
+%%LOGO%%
+
+# How to use this image
+
+The basic pattern for starting a `%%REPO%%` instance is:
+
+```console
+$ docker run --name some-%%REPO%% -d %%REPO%%
+```
+
+If you'd like to be able to access the instance from the host without the container's IP, standard port mappings can be used:
+
+```console
+$ docker run --name some-%%REPO%% -p 8080:80 -d %%REPO%%
+```
+
+Then, access it via `http://localhost:8080` or `http://host-ip:8080` in a browser.
+
+There are multiple database types supported by this image, most easily used via standard container linking. In the default configuration, SQLite can be used to avoid a second container and write to flat-files. More detailed instructions for different (more production-ready) database types follow.
+
+When first accessing the webserver provided by this image, it will go through a brief setup process. The details provided below are specifically for the "Set up database" step of that configuration process.
+
+## MySQL
+
+```console
+$ docker run --name some-%%REPO%% --link some-mysql:mysql -d %%REPO%%
+```
+
+-	Database type: `MySQL, MariaDB, or equivalent`
+-	Database name/username/password: `<details for accessing your MySQL instance>` (`MYSQL_USER`, `MYSQL_PASSWORD`, `MYSQL_DATABASE`; see environment variables in the description for [`mariadb`](https://registry.hub.docker.com/_/mariadb/))
+-	ADVANCED OPTIONS; Database host: `mysql` (for using the `/etc/hosts` entry added by `--link` to access the linked container's MySQL instance)
```

Comment:
According to the `--link` above, this should be `some-mysql` instead of just `mysql`.